### PR TITLE
add variant sku to the OCST report

### LINKED ISSUE #10030: Add Variant SKU to Order Cycle Supplier Totals report
## Description

As an enterprise user, I want to be able to access variant SKUs from the reports.
Let's add it to the Order Cycle Supplier Totals report

## Acceptance Criteria & Tests

1. Variant SKU added to the line item rows of Order Cycle Supplier Totals report
2. To be added to the end of each row.
3. These columns to be hidden by default in UI. API will return all data 'columns'.
4. All fields translatable
5. Unit test to cover this functionality.

### PR DESCRIPTION
**Pull request by abdellani**
#### What? Why?

- Closes #10030

#### What should we test?

- described on #10030

#### Release notes

Changelog Category: User facing changes

### CODE REVIEW

File: lib/reporting/reports/orders_and_fulfillment/order_cycle_supplier_totals.rb [hunk 1]
```diff
@@ -41,6 +42,10 @@ def query_result
             [e.variant_id, e.price]
           }.values
         end
+
+        def default_params
+          super.merge({ fields_to_hide: ["sku"] })
```
Comment (jibees) [inline lib/reporting/reports/orders_and_fulfillment/order_cycle_supplier_totals.rb:47]: 👌

**Review by jibees — APPROVED** [review]
Thanks! 🙏

**Comment by drummer83** [comment]
Hi @abdellani,
Thanks for yet another one! :muscle: 

I have tested this according to the acceptance criteria from the original issue #10030:
- Variant SKU added to the line item rows of Order Cycle Supplier Totals report. :heavy_check_mark: 
- To be added to the end of each row. :heavy_check_mark: 
image

- These columns to be hidden by default in UI. :heavy_check_mark: 
image

- API will return all data 'columns'.
image

- All fields translatable. :heavy_check_mark: (no new strings introduced)
- Unit test to cover this functionality. :heavy_check_mark: 

Additional tests:
- Other reports remain unchanged. :heavy_check_mark: 
- Tested as admin as well as super admin user. :heavy_check_mark: 
- Activating and deactivating the summary and header rows. :heavy_check_mark: 
- Tested that the SKU is included in pdf, xlsx and csv as well. :heavy_check_mark: 
image

Cool, I didn't find any problems here! Well done! This is ready to go! :rocket: 
Thanks!

**Comment by audez** [comment]
thankssss <3

**Pull request merged**